src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends ColorFigure {
+    private double radiusSquare;
+
+    public Circle(String color, double radiusSquare) {
+        super(color);
+        this.radiusSquare = radiusSquare;
+    }
+
+    public double getRadiusSquare() {
+        return radiusSquare;
+    }
+
+    public void setRadiusSquare(double radiusSquare) {
+        this.radiusSquare = radiusSquare;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: , circle:  "
+                + getArea() + " sq. units, " + "radiusSquare: "
+                + radiusSquare + " units, color: " + getColor());
```

Comment:
The `draw` method's output is missing the figure type. Consider adding 'circle' to the printed statement to clarify the type of figure being drawn.